### ISSUE TITLE
Lada's @email-address comment

### ISSUE THREAD

**Issue by kwatsen** [netconf-wg/list-pagination#11]
Lada's wrote:
```
- sec A.3.6.2 (and other places)
 Why "@email-address"? I think it should be "email-address" (though with
 namespace prefix, see above), because "email-address" is a normal YANG leaf
 represented as XML element in instance data. Or am I missing something?
```

Also, should draft say anything about querying on metadata?

**Comment by kwatsen** [comment]
Simply remove the '@' symbol?

**Comment by avtobiff** [comment]
Fixed in f8a52de
